sdk/eventhub/testhub/package.json

```diff
@@ -32,8 +32,8 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/event-hubs": "^2.1.1",
-    "@azure/event-processor-host": "^2.0.0",
+    "@azure/event-hubs": "^5.0.0-preview.6",
```

Comment:
`test hub` is not updated to use the latest event hubs yet.
There is an issue tracing the future of this package